wiki/Contests/Official_support/en.md

```diff
@@ -24,13 +24,37 @@ Officially supported contests have access to certain benefits:
 - **Progress towards *Elite Mapper* user title.** See [contest points](/wiki/Contests/Contest_points) for more details.
 - **Potential consideration for [news coverage](#news-posts)**.
 
+## Expectations
+
+Contest organisers are expected to monitor their contest for anything that runs afoul of the expectations outlined below, and are single-handedly responsible for reporting such incidents to the account support team via [tournaments@ppy.sh](mailto:tournaments@ppy.sh). Failure to adhere to this may result in withdrawal of official support.
+
+### Contests
+
+Community-run contests receiving this support are expected to abide by the following general ideals:
+
+- Remain free and fair.
+- Be as transparent as possible.
+- Preserve competitive integrity.
+- Encourage good sportsmanship.
+
+In addition, all promotional material or any services associated with a contest receiving official support should adhere to the [osu! community rules](/wiki/Rules). This includes things like Twitter accounts, Discord servers, and so on.
+
+### Additional expectations
+
+While the following expectations mainly apply to tournaments, contests are still expected to abide by them to the best of their ability:
+
+- [Staff](/wiki/Tournaments/Official_support#staff)
+- [Players](/wiki/Tournaments/Official_support#players)
+
 ## Eligibility
 
 Contests which abide by the following criteria are eligible for official support for **profile badges**:
 
 - **The contest must meet either of these criteria:**
   - The contest is organised and run by an experienced team of accomplished and reputable staff, as decided by the osu! team.
   - The contest has successfully run for one iteration without official support.
+- **The forum threads associated with the contest MUST contain a clearly visible and accessible link to the contest reports form.** The link must be placed at the bottom of the original post.
```

Comment:
```suggestion
- **The forum threads associated with the contest MUST contain a clearly visible and accessible link to the contest report form.** The link must be placed at the bottom of the original post.
```

Reply:
for some words plural sounds more natural inside compounds (like results), but i can't find "reports form" used anywhere else on the internet. generally, words inside compound terms aren't pluralised

Reply:
"The form for contest reports" rather than "The form to report a contest" that's how I read the difference between the one and another.

In any case, if it's changed here, may as well change it in the other. At the end of the day it's just 1 (one) letter, so I do not mind it either way.

Reply:
"report form" flows more naturally imo